File: JavaAthletic/RestServices/src/main/java/rest_domain/Competition.java
package rest_domain;

import domain.Entity;

public class Competition extends Entity<Long> {
    private String event;
    private String ageGroup;

    public Competition() {}

    public Competition(String event, String ageGroup) {
        this.event = event;
        this.ageGroup = ageGroup;
    }

    public String getEvent() {
        return event;
    }

    public void setEvent(String event) {
        this.event = event;
    }

    public String getAgeGroup() {
        return ageGroup;
    }

    public void setAgeGroup(String ageGroup) {
        this.ageGroup = ageGroup;
    }

    @Override
    public String toString() {
        return "ID: " + this.getId() + " | Event: " + this.getEvent() + " | Age Group: " + this.getAgeGroup();
    }

}

File: CSharpAthletic/RestServices/Competition.cs
namespace RestServices
{
    public class Competition
    {
        public long id { get; set; }
        public string @event { get; set; }
        public string ageGroup { get; set; }

        public override string ToString()
        {
            return "ID: " + id + " | Event: " + @event + " | Age Group: " + ageGroup;
        }
    }
}
